File: LeetCode/Easy/Convert Date to Binary/3280.js
/**
 * @param {string} date
 * @return {string}
 */
var convertDateToBinary = function(date) {
    const year = (+date.substring(0, 4)).toString(2);
    const month = (+date.substring(5, 7)).toString(2);
    const day = (+date.substring(8)).toString(2);
    return `${year}-${month}-${day}`;
};

File: LeetCode/Easy/Convert Date to Binary/3280.java
class Solution {
    public String convertDateToBinary(String date) {
        String year = Integer.toString(Integer.parseInt(date.substring(0, 4)), 2);
        String month = Integer.toString(Integer.parseInt(date.substring(5, 7)), 2);
        String day = Integer.toString(Integer.parseInt(date.substring(8)), 2);
        return year + "-" + month + "-" + day;
    }
}
